dds/DCPS/JobQueue.cpp

```diff
@@ -33,6 +34,7 @@ int JobQueue::handle_exception(ACE_HANDLE /*fd*/)
   }
 
   for (Queue::const_iterator pos = q.begin(), limit = q.end(); pos != limit; ++pos) {
+    ThreadStatusManager::Event ev(thread_status_manager);
```

Comment:
Windows is warning about the shadowed declaration here, should there be two of them?

Reply:
It is supposed to have both "events" since they support nesting.  However, now that I look at the code again in ThreadStatusManager, it needs to store the timestamp even for a nested update.
And then I'll change the variable name to fix the warning.